dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx

```diff
@@ -0,0 +1,286 @@
+import { css } from '@emotion/react';
+import { brandAlt, from, neutral, space } from '@guardian/source/foundations';
+import { LinkButton, SvgGuardianLogo } from '@guardian/source/react-components';
+import { useEffect, useState } from 'react';
+import {
+	removeMediaRulePrefix,
+	useMatchMedia,
+} from '../../../../lib/useMatchMedia';
+import { ThreeTierChoiceCards } from '../../epics/ThreeTierChoiceCards';
+import type { SupportTier } from '../../epics/utils/threeTierChoiceCardAmounts';
+import type { ReactComponent } from '../../lib/ReactComponent';
+import { bannerWrapper, validatedBannerWrapper } from '../common/BannerWrapper';
+import type { BannerRenderProps } from '../common/types';
+import { DesignableBannerArticleCount } from './components/DesignableBannerArticleCount';
+import { DesignableBannerBody } from './components/DesignableBannerBody';
+import { DesignableBannerCloseButton } from './components/DesignableBannerCloseButton';
+import { DesignableBannerHeader } from './components/DesignableBannerHeader';
+import type { BannerTemplateSettings } from './settings';
+import { templateSpacing } from './styles/templateStyles';
+
+const DesignableBannerV2: ReactComponent<BannerRenderProps> = ({
+	content,
+	countryCode,
+	onCloseClick,
+	articleCounts,
+	separateArticleCount, // legacy field
+	separateArticleCountSettings,
+	submitComponentEvent,
+	design,
+	onCtaClick,
+}: BannerRenderProps): JSX.Element => {
+	const isTabletOrAbove = useMatchMedia(removeMediaRulePrefix(from.tablet));
+
+	const [iosAppBannerPresent, setIosAppBannerPresent] = useState(false);
+
+	const [
+		threeTierChoiceCardSelectedProduct,
+		setThreeTierChoiceCardSelectedProduct,
+	] = useState<SupportTier>('SupporterPlus');
+
+	const variantOfChoiceCard = 'TWO_TIER_CHOICE_CARDS'; //this may need changed when US involved
+
+	useEffect(() => {
+		setIosAppBannerPresent(
+			window.innerHeight !== window.document.documentElement.clientHeight,
+		);
+	}, []);
+
+	useEffect(() => {
+		if (iosAppBannerPresent) {
+			if (submitComponentEvent) {
+				submitComponentEvent({
+					component: {
+						componentType: 'ACQUISITIONS_OTHER',
+						id: 'safari-ios-banner-present',
+					},
+					action: 'VIEW',
+				});
+			}
+		}
+	}, [iosAppBannerPresent, submitComponentEvent]);
+
+	if (!design) {
+		return <></>;
+	}
+
+	//All config hard coded for the tests - TODO make this configurable in the future
```

Comment:
Trello card to cover this TODO https://trello.com/c/sLa1GEg7/829-3-tier-banners-wire-in-config-to-tooling